lib/opentok/client.rb

```diff
@@ -164,6 +164,28 @@ def delete_archive(archive_id)
       raise OpenTokError, "Failed to connect to OpenTok. Response code: #{e.message}"
     end
 
+    def layout_archive(archive_id, opts)
+      opts.extend(HashExtensions)
+      response = self.class.put("/v2/project/#{@api_key}/archive/#{archive_id}/layout", {
+          :body => opts.camelize_keys!.to_json,
+          :headers => generate_headers("Content-Type" => "application/json")
+      })
+      case response.code
+      when 200
+        response
+      when 400
+        raise OpenTokArchiveError, "Setting the layout failed.The request was invalid or maybe invalid layout options were given."
```

Comment:
@msach22 should we use the same error messages that are described in the documentation? https://tokbox.com/developer/rest/#change_composed_archive_layout

Reply:
Yes, let's use the response that's the same as the API server

Reply:
I think @JayTokBox is following the error handling that the SDK has been following so I think it's okay. We should clean up all SDKs to simply forward the REST API response.

Reply:
Ok, sounds good @msach22 , so I will merge this PR after the last check